pf4j/src/test/java/org/pf4j/SingletonExtensionFactoryTest.java

```diff
@@ -74,7 +74,7 @@ public void createNewEachTimeFromDifferentClassLoaders() throws Exception {
     private URL[] getClasspathReferences() throws MalformedURLException {
         String classpathProperty = System.getProperty("java.class.path");
 
-        String[] classpaths = classpathProperty.split(":");
+        String[] classpaths = classpathProperty.split(";");
```

Comment:
I don't know what to say here. Probably the correct approach is to use `File.pathSeparator`.
The `;` character is used to separate the file paths on Windows.